deploy.go

```diff
@@ -49,9 +49,10 @@ type deployOptions struct {
 	AddEFIBootManagerProfile bool   `long:"add-efi-boot-manager-profile" description:"Protect the disk unlock key with the EFI boot manager code and boot attempts profile (PCR4)"`
 	AddEFISecureBootProfile  bool   `long:"add-efi-secure-boot-profile" description:"Protect the disk unlock key with the EFI secure boot policy profile (PCR7)"`
 	AddUbuntuKernelProfile   bool   `long:"add-ubuntu-kernel-profile" description:"Protect the disk unlock key with properties measured by the Ubuntu kernel (PCR12). Also prevents access outside of early boot"`
-
-	AzDiskProfile string `long:"az-disk-profile" description:""`
-	UefiConfig    string `long:"uefi-config" description:"JSON file describring the platform firmware configuration"`
+	RootPartitionUUID        string `long:"root-partition-uuid" description:"Explicitly provide the root partition UUID instead of relying on partition type"`
+	ESPPartitionUUID         string `long:"efi-partition-uuid"  description:"Explicitly override the ESP partition UUID instead of relying on partition type"`
```

Comment:
Redundant space before `description`

`Explicitly provide` instead of `Explicitly override` (for consistency with the line above)